addons/arsenal/arsenale/fnc_arsenalUSA.sqf

```diff
@@ -903,7 +912,8 @@ _items append (switch (ACE_player getVariable ["TB_rolle", ""]) do
             "rhs_weap_sr25_ec_d",
             "rhs_weap_sr25_ec_wd",
             "rhs_weap_sr25_wd",
-
+            "rhs_weap_XM2010",
+            "rhs_weap_XM2010_d",
```

Comment:
👀

Reply:
Um ehrlich zu sein, weiß ich nur noch, dass ich von 2 Leuten Beschwerden bekommen hatte, dass es kein 7.62mm mit Bolt Action mehr gäbe. Ich selbst halte nicht viel davon, da es nicht praktikabel ist; es ginge wohl um das Gefühl...
Das war Mordl zum einen und die andere Person weiß ich nicht mehr, kann aber nachfragen.